src/cdk-experimental/listbox/listbox.spec.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  ComponentFixture,
+  async,
+  TestBed,
+} from '@angular/core/testing';
+import {Component, DebugElement} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {
+  CdkOption,
+  CdkListboxModule
+} from './index';
+import {dispatchMouseEvent} from '@angular/cdk/testing/private';
+
+describe('CdkOption', () => {
+
+  describe('selection state change', () => {
+    let fixture: ComponentFixture<CdkListboxWithCdkOptions>;
+    let options: DebugElement[];
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkListboxModule],
+        declarations: [CdkListboxWithCdkOptions],
+      }).compileComponents();
+    }));
+
+    beforeEach(async(() => {
+      fixture = TestBed.createComponent(CdkListboxWithCdkOptions);
+      fixture.detectChanges();
+      options = fixture.debugElement.queryAll(By.directive(CdkOption));
+    }));
+
+    it('should generate a unique optionId for each option', () => {
+      for (const option of options) {
+        expect(option.injector.get<CdkOption>(CdkOption).getOptionId()).toMatch(/cdk-option-\d+/);
```

Comment:
optional: you could also test that all of the IDs are different

Reply:
In making this test, I realized the .includes() method for arrays isn't supported. I think it came with ES7. Is this expected for the current Angular version?